charts/timescaledb-single/Chart.yaml

```diff
@@ -4,7 +4,7 @@
 apiVersion: v1
 name: timescaledb-single
 description: 'TimescaleDB HA Deployment.'
-version: 0.30.0
+version: 0.31.0
```

Comment:
Sorry this now needs to be `0.33.0`.  There have been a few merges since my last comment.